core-services/release-controller/_releases/release-ocp-4.16.json

```diff
@@ -546,6 +546,12 @@
         "name": "periodic-ci-openshift-release-master-nightly-4.16-e2e-vsphere-ovn-techpreview-serial"
       }
     },
+    "vsphere-ovn-uprade": {
+      "optional": true,
```

Comment:
I think you need to add ` "upgrade": true` also need to add for release-ocp-4.17.json